scene/gui/option_button.cpp

```diff
@@ -203,16 +203,18 @@ void OptionButton::pressed() {
 }
 
 void OptionButton::add_icon_item(const Ref<Texture2D> &p_icon, const String &p_label, int p_id) {
+	bool first_selectable = !has_selectable_items();
 	popup->add_icon_radio_check_item(p_icon, p_label, p_id);
-	if (popup->get_item_count() == 1) {
-		select(0);
+	if (first_selectable) {
+		select(get_item_count() - 1);
```

Comment:
I think get_item_count() can be zero?

Reply:
In here, it means an item has just been added, so it wont be zero.